lib/rmt/cli/decorators/base.rb

```diff
@@ -13,8 +13,13 @@ def to_table
 
   protected
 
-  def array_to_csv(array, headers)
-    CSV.generate { |csv| array.unshift(headers).each { |row| csv << row } }
+  def headers_to_csv(headers)
+    CSV.generate { |csv| csv << headers }
+  end
+
+  def array_to_csv(array, headers, batch: nil)
```

Comment:
On `lib/rmt/cli/decorators/system_decorator.rb` you use `batch: false` to set the default value, I think it would be nice to keep it standardized across the code.